src/services/api.service.ts

```diff
@@ -149,13 +149,20 @@ export const getSeriesByMediaIds = async (mediaIds: string[]): Promise<{ [mediaI
  * Get all episodes of the selected series (when no particular season is selected or when episodes are attached to series)
  * @param {string} seriesId
  */
-export const getEpisodes = async (seriesId: string | undefined, pageOffset: number, pageLimit?: number): Promise<EpisodesWithPagination> => {
+export const getEpisodes = async (
+  seriesId: string | undefined | null,
+  pageOffset: number | null,
+  pageLimit: number | null = PAGE_LIMIT,
+  afterId?: string | null,
+): Promise<EpisodesWithPagination> => {
   if (!seriesId) {
     throw new Error('Series ID is required');
+  } else if (pageOffset && afterId) {
```

Comment:
I would suggest we either let the BE handle this or enforce it with typings on the arguments:
```
args: {seriesdId: string} & ({pageOffset: number, pageLimit: number} | {afterId: string})
```

Reply:
Removed this check